routes/web.php

```diff
@@ -12,13 +12,15 @@
 */
   Route::get('/', function () {
       return view('welcome');
-  });
+  })->middleware('guest');
```

Comment:
yeah that sounds, good. I'm good with whatever you guys choose, I just want to make sure I know what you guys are doing so I can adhere to the same conventions